src/Donors/ServiceProvider.php

```diff
@@ -18,6 +19,12 @@ class ServiceProvider implements ServiceProviderInterface
     public function register()
     {
         give()->singleton('donors', DonorRepositoryProxy::class);
+        give()->singleton(DonorsListTable::class, function() {
+            $listTable = new DonorsListTable();
+            do_action('give_donors_list_table', $listTable);
```

Comment:
Is this a legacy action we are supporting or introducing a new one?

Reply:
Resolved: 6aa6c99d0d9b74e57f5f84ff7201421403aff660